css-two-column-layout/styles.css

```diff
@@ -0,0 +1,83 @@
+/* A new css rule set selecting all elements in the 'body' elements ... Opening curly bracket */
+.wrapper-container {
+  /* the code below is to get all items in contain to be horizontally centered in browser */
+  margin: 0 auto; /* centering body */
+
+  /* the code below is to get all items in contain to be vertically centered in browser */
+  display: flex;
+  justify-content: center; /* horizontal effect on items of 'container' class */
+  align-items: center; /* vertical effect on items of 'container' class */
```

Comment:
You don't really want to apply `display: flex` on your `.container`. This should be applied to your `row` elements. Your container just acts as a wrapper for the full page. We don't want all of our page to be on one line, we just want each row to be on a line